e2e/libs/longhorn_deploy/longhorn_deploy.py

```
@@ -18,11 +18,29 @@ def __init__(self):
     def check_longhorn_crd_removed(self):
         return self.longhorn.check_longhorn_crd_removed()
 
-    def install(self, is_stable_version=False):
-        return self.longhorn.install(is_stable_version)
+    def install(self, install_stable_version):
+        self.longhorn.create_longhorn_namespace()
+        self.longhorn.install_backupstores()
+        self.longhorn.create_registry_secret()
+        installed = self.longhorn.install(install_stable_version)
+        if not installed:
+            logging(f"Installing Longhorn failed")
```

Ruff (0.8.2)

Undefined name `logging`

(F821)